components/Footer.js

```diff
@@ -1,7 +1,95 @@
 import React from "react";
+import {
+	FaDiscord,
+	FaInstagram,
+	FaSlack,
+	FaYoutube,
+	FaMediumM,
+	FaGithub,
+} from "react-icons/fa";
 
 const Footer = () => {
-	return <div>Footer</div>;
+	return (
+		<footer className='bg-neutral-900 text-gray-50 py-6'>
+			<div className='grid sm:grid-rows-2 justify-center mx-auto'>
+				<div>
+					<p className='text-4xl font-lexend font-bold text-center'>
+						Let&#39;s connect!
+					</p>
+				</div>
+				<div>
+					<div className='grid sm:grid-cols-6 grid-cols-3 justify-items-center mx-auto gap-4'>
+						<div>
```

Comment:
What are these div's for? The ones without a class name, you can remove them and just have the a tag. Less code -> lighter website